packages/cli/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@sigstore/cli",
+  "version": "0.0.1",
+  "description": "Sigstore CLI",
+  "author": "Brian DeHamer @bdehamer",
+  "license": "Apache-2.0",
+  "keywords": [
+    "cli",
+    "sigstore"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "bin": {
+    "sigstore": "./bin/run"
+  },
+  "files": [
+    "/bin",
+    "/dist",
+    "/oclif.manifest.json"
```

Comment:
Does this manifest need to be checked in?

Reply:
No, it's generated as part of the release process, packaged in the tarball and then immediately deleted.

Really we could omit the manifest altogether -- I believe it's only useful if we wanted this CLI to function as a plugin to some other CLI.